WeakAuras/WeakAuras.lua

```diff
@@ -2961,6 +2961,27 @@ function WeakAuras.Modernize(data)
     end
   end
 
+  -- Version 17 & 18 were introduced in June 2019 in BFA
```

Comment:
if these are in the same submission, then they can be in the same internalVersion, by the way.

Reply:
yup i'm going to change it